db/people.yml

```diff
@@ -3800,10 +3800,21 @@ yztao:
   position: post-doctoral scholar
 zcliu:
   active: true
-  aka: []
+  aka:
+    - Zicheng Liu
+    - Taylor Liu
   avatar: https://github.com/zichengliu1997.png
-  bio: bio
-  education: []
+  bio: Zicheng is from Guangzhou, China. He has spent the last year working as an undergraduate
+    rsearch assistant in the plasma physics laboratory at Columbia University while majoring
+    in applied physics. He joined Dr. Billinge's group in the summer of 2020 as a visiting
+    undergraduate research assistant. For his GitHub ID, use zichengliu1997.
+  education:
+    - begin_year: 2015
+      begin_month: 8
+      degree: BS in Applied Physics & BA in Pre-engineering
+      institution: Columbia University & Bard College at Simon's Rock
```

Comment:
I guess you were a 3-2 student.  In this case we need to separate these into two education entries, one for Columbia and one for Bard, but they can have the same dates (or not if you prefer).  Use ``columbiau`` which is an ID for Columbia.  For Bard make an id (maybe ``bardcatsimonsrock`` and make an entry in the institutions.yml database.

When you make these changes, don't close this PR and make a new one, but just make the changes, add and commit them, and then push to the same branch on origin and they will appear in this PR.